gap/ToolsDerivedMethods.gi

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# FinSetsForCAP: The elementary topos of (skeletal) finite sets
+#
+# Implementations
+#
+
+##
+AddDerivationToCAP( ExactCoverWithGlobalElements,
+        [ [ MorphismsOfExternalHom, 1 ],
+          [ TerminalObject, 1 ] ],
+        
+  function ( cat, A )
+    
+    return MorphismsOfExternalHom( cat, TerminalObject( cat ), A );
+    
+end :
+  CategoryFilter := cat ->
+                    IsCategoryOfFinSets( cat ) or
+                    IsCategoryOfSkeletalFinSets( cat ) );
```

Comment:
If a "derivation" matches only a finite number of concrete primitive categories, we usually simply install it primitively. So I suggest doing that in this case, too.

Reply:
Done.